packages/node/src/tasks/is-eligible.ts

```diff
@@ -105,6 +105,7 @@ export class EligibilityTask extends EventEmitter<EventMap> {
 					logger.error(`Failed signing message for eligibility: ${messageSignature.error}`, {
 						id: traceId,
 					});
+					// TODO: Discuss how do we handle this ERROR for alerting & monitoring.
```

Comment:
CRITICAL: Identity tried to sign with non existent key. Something is serious wrong (This should never happen)